Heat_method_3/include/CGAL/Heat_method_3/Surface_mesh_geodesic_distances_3.h

```diff
@@ -29,6 +29,7 @@
 #include <CGAL/Eigen_solver_traits.h>
 #endif
 
+#include <CGAL/Weights/utils.h>
```

Comment:
Why do we need this header file. This is only ok, if the file is documented.

Reply:
It is documented. It includes functions such as cotangent() and tangent(), which are often used on their own.

Reply:
It should be similar to the header file [`CGAL/mesh_segmentation.h>`](https://github.com/CGAL/cgal/blob/master/Surface_mesh_segmentation/include/CGAL/mesh_segmentation.h#L19)

Reply:
Where is this file documented?  Please give me the link to the html documentation.

Reply:
It supposes to be [here](https://cgal.geometryfactory.com/~danston/Weights/group__PkgWeightsRefUtils.html) but since we document all weights in PackageDescription.txt the proper headers are not shown for all weights in fact. I have just noticed that. Any idea how to force doxygen to show the header where the function is placed?

Reply:
Why are these not functions in the CGAL kernel?  Because they call trigonomtric functions. @MaelRL @sloriot

Reply:
As discussed in element, just create an issue (feature request) with a link to the file,

Reply:
I have created issue #5814. So, we can move the discussion there.